backend/src/main/kotlin/com/pk/zbtz/zbtzbackend/databases/postgres/services/HumanService.kt

```diff
@@ -0,0 +1,32 @@
+package com.pk.zbtz.zbtzbackend.com.pk.zbtz.zbtzbackend.databases.postgres.services
+
+import com.pk.zbtz.zbtzbackend.com.pk.zbtz.zbtzbackend.databases.postgres.models.HumanPostgresModel
+import com.pk.zbtz.zbtzbackend.com.pk.zbtz.zbtzbackend.databases.postgres.repositories.HumanPostgresRepository
+import com.pk.zbtz.zbtzbackend.controllers.people.requests_and_responses.AddHumanRequest
+import org.springframework.stereotype.Service
+
+@Service
+class HumanService(
+    private val repository: HumanPostgresRepository
+) {
+    fun getById(id: Int): HumanPostgresModel? {
+        return repository.findById(id).orElseThrow()
+    }
```

Comment:
I think we can change to a nonthrowing Exception approach because the return value of this function could be null
```suggestion
    fun getById(id: Int): HumanPostgresModel? {
        return repository.findById(id)..getOrNull()
    }
```

Reply:
this way we could easily return 404 by catchingi n the global exception handler, we will not implement it because we don't have time for that so we are getting 500